data/examples/declaration/value/function/if-multi-line-out.hs

```diff
@@ -10,7 +10,6 @@ bar x =
     then
       foo x
         + 100
-    else
-      case x of
-        1 -> 10
-        _ -> 20
+    else case x of
```

Comment:
I think another example showing hanging on both `then` and `else` would be good.